### DISCUSSION TITLE
How do I change DAC to TM8211?

### DISCUSSION THREAD

**Discussion by EASYBOARD-CN** [schreibfaul1/ESP32-audioI2S#899] [Q&A]
Hello guys!
I could run I2Saudio.ino normally, use MAX98357 to play MP3. However I would like to change the MAX98357 to TM8211, just wondering where can I modify it? and how do I modify it?
Thx guys

**Comment by schreibfaul1** [comment]
DACs such as PT8211 and TM8211 are rarely used.
These use the LSBJ format (Least Significant Bit Justified).
I found a bug. Now it should work, please use an Arduino version &gt; V3.0.0 and then set `audio.setI2SCommFMT_LSB(true);`
